packages/strapi-admin/services/permission.js

```diff
@@ -180,6 +189,43 @@ const cleanPermissionInDatabase = async () => {
   }
 };
 
+const ensureBoundPermissionsInDatabase = async () => {
+  if (strapi.EE) return;
+
+  const models = Object.values(strapi.contentTypes);
+  const role = await strapi.query('role', 'admin').findOne({ code: EDITOR_CODE }, []);
+
+  for (const model of models) {
+    const actions = getBoundActionsBySubject(role, model.uid);
+    const perms = await strapi.query('permission', 'admin').find(
```

Comment:
Not sure, but If we didn't use an abbreviation keep permissions instead of perms